providers/implementations/ciphers/ciphercommon.c

```diff
@@ -169,6 +183,8 @@ int cipher_generic_dinit(void *vctx, const unsigned char *key, size_t keylen,
                                         iv, ivlen, 0);
 }
 
+#define MAX_PADDING 256
```

Comment:
If this includes the length then maybe comment on it..

Reply:
Commment added